forge/test/mlir/test_loss.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+
+import torch
+from torch import nn
+import forge
+from forge.op.eval.common import compare_with_golden
+
+
+@pytest.mark.parametrize(
+    "prediction_shape",
+    [
+        (3, 5),
+        (32, 32),
+        (33, 127),
+        (128, 20),
+    ],
+)
+@pytest.mark.push
+def test_l1_loss(prediction_shape):
+    forge_loss = forge.op.loss.L1Loss("l1_loss")
+    torch_loss = torch.nn.L1Loss()
+
+    prediction = torch.randn(prediction_shape, requires_grad=True)
+    prediction_forge = forge.tensor.Tensor.create_from_torch(prediction)
+    target = torch.empty(prediction_shape[0], dtype=torch.long).random_(prediction_shape[1])
+    target = nn.functional.one_hot(target, num_classes=prediction_shape[1]).float()
+    target_forge = forge.tensor.Tensor.create_from_torch(target)
+
+    forge_loss = forge.compile(forge_loss, sample_inputs=[prediction_forge, target_forge])
+    forge_loss_out = forge_loss(prediction, target)
+    torch_loss_out = torch_loss(prediction, target)
+
+    assert torch.allclose(torch_loss_out, forge_loss_out[0], rtol=11e-3)
```

Comment:
@vkovinicTT once you merge your PR, please checkout if there are new tests that are not incorporating your verification feature.

Also, once merged, make sure to post the details on #fore-fe channel, in order to communicate further for everyone to use this function for future tensor verifications.